### ISSUE TITLE
peerDependency is not compat with sinon@8

### ISSUE THREAD

**Issue by AviVahl** [domenic/sinon-chai#143]
`sinon@8` got released yesterday, and `sinon-chai` specifies a lower peerDependency:
```
warning " > sinon-chai@3.3.0" has incorrect peer dependency "sinon@>=4.0.0 <8.0.0".
```

FYI @domenic @simondel

**Comment by simondel** [comment]
Thanks for the heads-up! If anyone can make a PR, I’lol merge and release it tomorrow

**Comment by AviVahl** [comment]
@simondel ask and you shall receive: https://github.com/domenic/sinon-chai/pull/144

**Comment by simondel** [comment]
Released as sinon-chai@3.4.0
